.env.demo

```diff
@@ -2,3 +2,5 @@ PORT="PORT WHERE SERVER IS RUNNING"
 MONGO_CONNECTION_STRING="mongodb+srv://<username>:<password>@cluster0.jsllulh.mongodb.net/<database>?retryWrites=true&w=majority"
 JWT_PRIVATE_KEY="JWT PRIVATE KEY FOR ENCRYPTION"
 JWT_EXP_DURATION="EXPIRE DURATION FOR JWT KEY ex 1800s, 1d"
+TWILIO_ACCOUNT_SID=AC0bb22eb3fd211743b07543c12c2f500e
```

Comment:
are these the actual keys or just examples?

Reply:
The actual keys